examples/react/README.md

```diff
@@ -0,0 +1,16 @@
+# Web3.js + React Demonstration dApp
+
+This project is a demonstration dApp built with [Web3.js](https://web3js.org/)
+and [React](https://react.dev/).
+
+- [Web3.js Docs](https://docs.web3js.org/)
+- [React Docs](https://react.dev/learn)
+
+This project was bootstrapped with
+[Create React App](https://github.com/facebook/create-react-app).
+
+## Getting Started
+
+Install the project dependencies with a dependency manager like npm or Yarn. Use
+`npm start` to start a local development server and navigate to
+http://localhost:3000 to view the dApp.
```

Comment:
Would be nice to update this.

Reply:
what would you like to update here? the example app's purpose is to allow for manual tests of the wallet rpc plugin with different extension wallets

Reply:
I created this PR https://github.com/web3/web3-wallet-rpc-utils/pull/10